README.md

```diff
@@ -60,9 +60,46 @@ the squashed commit also according to the convention.
 
 ## Translations
 
-- Check out [react-i18next](https://react.i18next.com/) and [i18next](https://www.i18next.com/) docs for more info.
+- Check out [nitrolib](https://github.com/kiwicom/nitrolib) docs for more info.
 - [DOCs for Phraseapp](https://phraseapp.com/docs/) - where the translations are stored for translators
-- List of all supported languages is defined by file `i18n/languages.js`
+- List of all supported languages is defined by file `src/translations/langInfos.js`
+
+### How to use translations in code
+
+- Use `Text` from [nitrolib](https://github.com/kiwicom/nitrolib) as `Trans` and don't confuse it with `Text` from [Orbit](https://github.com/kiwicom/orbit-components):
+
+Frome Nitrolib docs:
+
+> Make sure to wrap translation keys in the global `__` function!
+
+```js
+import Trans from "@kiwicom/nitro/lib/components/Text";
+```
+
+Props:
+```js
+type Props = {
+  t: string, // the translation key
+  values?: Values, // a map of placeholders and their values
+  html?: boolean, // make it 'true' if your translation string contains HTML elements
+};
+```
+
+Example:
+```js
+const Submit = () => (
+  <Button>
+    <Trans t={__("smartfaq.submit_button")} />
```

Comment:
I don't really like this TBH because it makes it harder to find things using full text search. For example how would I find all components that contain a specific word, say "Integration"? Now it's very easy, for example `grep -i "integration" -r src`.

After this change we will have to do a full text search on all translations, and then a full text search for each translation key the translation of which contains the specific word. That makes it quite hard to make project wide changes such as changing "bags" to "baggage" in every component.